packages/components/src/navigable-container/tabbable.tsx

```diff
@@ -30,6 +38,8 @@ export function TabbableContainer( { eventToOffset, ...props }, ref ) {
 		if ( eventToOffset ) {
 			return eventToOffset( evt );
 		}
+
+		return undefined;
```

Comment:
This is pedantic, but it prevents a warning from `innerEventToOffset()` sometimes not returning anything.